dashboard/app/helpers/proxy_helper.rb

```diff
@@ -5,6 +5,19 @@
 
 # Helper which fetches the specified URL, optionally caching and following redirects.
 module ProxyHelper
+  # https://www.iana.org/assignments/iana-ipv4-special-registry/iana-ipv4-special-registry.xhtml
+  PRIVATE_IPS = [
+    IPAddr.new('0.0.0.0/8'),
+    IPAddr.new('10.0.0.0/8'),
+    IPAddr.new('127.0.0.0/8'),
+    IPAddr.new('172.16.0.0/12'),
+    IPAddr.new('169.254.0.0/16'),
+    IPAddr.new('192.168.0.0/16'),
+    IPAddr.new('fd00::/8')
+  ].freeze
```

Comment:
I think this check could be simplified to `ip.private? || ip.link_local? || ip.loopback? || IPAddr.new('0.0.0.0/8').include?(ip)`, which would offload some of these details to the Ruby standard lib.

Reply:
I didn't notice those handy methods.  Switched to using them!